### ISSUE TITLE
Notion about reproducing this paper. 

### ISSUE THREAD

**Issue by Hzzone** [yoyohonyang/LearingFaceAgeProgression#10]
I have suffered a long time to reproduce similar results against this paper. Although I will not public the implementation, I am willing to share some experience that should be stressed on.
1. PAG-GAN are not cGAN-based methods, eg., CAAE and IPCGAN. 
```
In our case, the CNN based generator takes young faces as inputs, and learns a mapping to a domain corresponding to elderly faces.
```
That's to say, they trained a model for every mapping, ie., 30- >>> 31-40, 30- >>> 41-50 and so on..... It confused me for such a long time.....
2. The feature maps after the pre-trained VGG-Face and Age Classifier should include the activation function.

Hopefully, it helps your guys research....

**Comment by Hzzone** [comment]
More tricks I found will be followed.

**Comment by Hzzone** [comment]
I have achieved a similar quantitative result, although there is strong color distortion in generated images which is not  so good compared to the original paper. That's because of the use of BN and GAN Loss(the real young faces are considered as fake data). Besides, I have tried to remove the BN in D, and due to the tanh function, the gradient for G is saturated, leading to the weight difficult to be updated.

In summary, this work is really difficult to be reproduced, and I have never found a paper that has achieved a comparable result than the original paper. Hopefully, their training code would be public in the future.

**Comment by MaxiFilippov** [comment]
How much time did you spend to train it? And what gpu?

**Comment by MaxiFilippov** [comment]
What do you think is there any way to achieve sush result as FaceApp old filter?

**Comment by MaxiFilippov** [comment]
@Hzzone Good job! How much time did you spend to train it? And what gpu?

**Comment by Hzzone** [comment]
It should be very quick and takes me about one night with single 2080Ti.

**Comment by MaxiFilippov** [comment]
@Hzzone And what do you think is there a way to achieve such results as FaceApp old filter?

**Comment by MaxiFilippov** [comment]
@Hzzone Could you answer my question please?:) Can I just download .t7 file from this repo and then use it without making architecture?

**Comment by Hzzone** [comment]
No, the shared model file only translated 30- into 51+.

**Comment by MaxiFilippov** [comment]
@Hzzone OK, thank you. What do you think is there a way to achieve results as Face App old filters using public datasets?

**Comment by MaxiFilippov** [comment]
@Hzzone Hi:)
How to make the model convert to 70+ age?
